conf/formTemplateSchema.json

```diff
@@ -1175,22 +1175,50 @@
       }
     },
     "stringOrEnCyObject": {
-      "type": [
-        "string",
-        "object"
-      ],
-      "properties": {
-        "en": {
+      "oneOf": [
+        {
           "type": "string"
         },
-        "cy": {
-          "type": "string"
+        {
+          "type": "object",
+          "properties": {
+            "en": {
+              "type": "string"
+            },
+            "cy": {
+              "type": "string"
+            }
+          },
+          "required": ["en"],
+          "additionalProperties": false
+        },
+        {
+          "type": "array",
+          "minItems": 2,
```

Comment:
This limitation is quite misleading in terms of error message you get when you use this valid definition


```
"infoText": [
  {
    "en": "Abc",
    "cy": "Zyx"
  }
]
 ```
 
 I think it would be better to drop this constraint and do more precise validation in gform

Reply:
done